include/snowflake/client.h

```diff
@@ -265,6 +266,7 @@ typedef struct SF_CONNECT {
 
     char *passcode;
     sf_bool passcode_in_password;
+    sf_bool log_spec_info;
```

Comment:
NIT: I'm not sure if the name for `log_spec_info` is clear what it does.

Reply:
I think this attributes will be used widely when we plan to add more logs in libsnowflakeClient. Therefore, I choose this name means we need to add some specific logs in libsnowflakeClient. Please help me find some more meaningful name :)

Reply:
Hopefully we won't need to add attributes like this once libSfClient is ported to XP. So we can go with something more specific here, a name like `log_query_exec_steps_info` or anything that you can think of.